### PR TITLE
Corpus update

### PR DESCRIPTION
**Pull request by vkosuri**
This PR includes tests for corpus

### CODE REVIEW

**Comment by vkosuri** [comment]
@gunthercox Any comments on this PR?

**Comment by vkosuri** [comment]
Master could you please travis.ci for this repository?

File: tests/training_tests/test_ubuntu_corpus_training.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+import unittest
+import tarfile
+import os
+from mock import Mock
+
+from tests.base_case import ChatBotTestCase
+from chatterbot.trainers import UbuntuCorpusTrainer
+
+
+class UbuntuCorpusTrainerTestCase(ChatBotTestCase):
```
Comment (gunthercox) [inline tests/training_tests/test_ubuntu_corpus_training.py]: This test case doesn't appear to be for the ChatterBot corpus.

File: tests/training_tests/test_twitter_trainer.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        'get_search.json'
+    )
+    tweet_data = open(data_file)
+    data = json.loads(tweet_data.read())
+    tweet_data.close()
+
+    return Status.NewFromJsonDict(data.get('statuses')[1])
+
+
+class TwitterTrainerTestCase(ChatBotTestCase):
```
Comment (gunthercox) [inline tests/training_tests/test_twitter_trainer.py]: The Twitter training test case probably isn't relevant to the ChatterBot corpus data.

**Comment by vkosuri** [comment]
@gunthercox Master any comments/suggestions?

**Comment by vkosuri** [comment]
Thanks Master

**Pull request merged**